### DISCUSSION TITLE
Alternative trained pipelines installation steps

### DISCUSSION THREAD

**Discussion by blade1989** [explosion/spaCy#12796] [Help: Installation] [answered]
I am behind a secured network, where githuh is blocked.

I tried the installation steps for trained Pipelines:

```bash
python -m spacy download en_core_web_sm
```

But that just outputs the error:

```bash
ERROR: Could not install packages due to an OSError: HTTPSConnectionPool(host='github.com', port=443): Max retries exceeded with url: /explosion/spacy-models/releases/download/nl_core_news_sm-3.5.0/nl_core_news_sm-3.5.0-py3-none-any.whl (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1129)')))
```

What are the alternative installation steps for trained pipelines?

**Comment by svlandeg** [comment, marked as answer]
Hi!

We typically do rely on Github for hosting our models. If you can't use the `spacy download` command, you could download the model first (on a different system maybe?) and transfer it and install it directly.

Our [models](https://spacy.io/models/) documentation pages have a "release details" link where you can download the model directly, then install it like so :

```
pip install zh_core_web_trf-3.6.1.tar.gz
```

Alternatively, you can also find our models on the Huggingface hub: https://huggingface.co/spacy. While we usually maintain a mirror of the most recent releases there, it's not guaranteed to be stable, as officially we do host our models on Github.

**Comment by blade1989** [comment]
I cannot find the "release details" link?

**Reply by rmitsch** [reply]
It's on the top right of the center pane of the [model page](https://spacy.io/models/zh).
![image](https://github.com/explosion/spaCy/assets/7152225/b5709ae6-9d1b-4234-bf4a-3c84147b2d76)
